src/promptflow/promptflow/azure/operations/_flow_operations.py

```diff
@@ -388,7 +389,7 @@ def list(
 
     def _download(self, source, dest):
         # TODO: support download flow
-        raise NotImplementedError("Not implemented yet")
```

Comment:
Why do we need to use `NotImplementedErrorException` here? Not sure if all devs can realize they need to throw this for not implemented error.

Reply:
Because these errors will also be flagged in the telemetry, and the error category is UNKNOW.
We cannot demand that all dev be written according to specifications, we can only to add tests to cover.
![image](https://github.com/microsoft/promptflow/assets/23182548/c3476238-bbf2-42c6-bcd3-ae05513185d4)